setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='robot',
-    version='1.0.0',
```

Comment:
I believe we agreed the version number would also contain the year, ie `2018.2.0` (as this is second release this year)

Reply:
fixed in 8970025 (which I now realise has a typo in the commit message)